### ISSUE TITLE
SA1001 incorrectly reported at beginning of line

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#257]
The diagnostic for SA1001 does not correctly determine whether a token is the first in the line or not. The following code produces an incorrect SA1001 warning for the `,` token. Note that this is only reproduced when the code is not indented.

```csharp
int[] data = { 3
, 2 };
```
